python/whylogs/core/feature_weights.py

```diff
@@ -1,11 +1,11 @@
 import json
-from typing import Any, Dict, Tuple, Union
+from typing import Any, Dict, List, Optional, Tuple, Union
 
-from whylogs.api.writer import Writers
-from whylogs.api.writer.writer import Optional, Writable, Writer
+from whylogs.api.writer.writer import Writable
 from whylogs.core import Segment
 
 
+# TODO: In whlogs 2.X this should not be a Wriable, it should be sent via WhyLabsClient
```

Comment:
If we're removing this way of uploading feature weights, what is the recommended way? Can we have an integration test showing the suggested way?

Also, what is the rationale for feature weights not being a Writable? I'm not opposed to this, just curious.

Reply:
Recommended way is `WhyLabsClient::write_feature_weights()`. I will add the integration test.

The rationale is that feature weights are specific to WhyLabs rather than being whylogs data.